### PR TITLE
Add Ohio CDCC and senior citizen credit

### PR DESCRIPTION
**Pull request by Linfeng-ll**
adding child and dependent care credits and senior citizen credits.

### CODE REVIEW

**Review by nmrodelo — CHANGES_REQUESTED** [review]
Added formatting changes, updated changelog to reflect that two variables are being added ( we forgot to add senior citizen credit as its own PR). Unable to update version on my end, @MaxGhenis could you get that? Alternatively, @Linfeng-ll , if you have time before we meet tonight to update that would be great.

File: =0.1.7 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Requirement already satisfied: yaml-changelog in /opt/conda/envs/policyengine/lib/python3.9/site-packages (0.3.0)
```
Comment (MaxGhenis) [inline =0.1.7:1]: remove file

**Review by MaxGhenis — COMMENTED** [review]
is this ready for review? if so please mark it as such (currently in draft) and i'll take a closer look

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_upper_limit.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+description: Ohio CDCC AGI Limit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_upper_limit.yaml:1]:
```suggestion
description: Ohio limits its CDCC amount at this AGI threshold.
```

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_upper_limit.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+values:
+  2021-01-01: 40_000
+metadata:
+  unit: currency-USD
+  period: year
+  label: OH Child Care and Dependent Care Credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_upper_limit.yaml:7]:
```suggestion
  label: OH Child Care and Dependent Care Credit income threshold
```

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_upper_limit.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  reference:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_upper_limit.yaml:8]: add legal code reference

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/child/cdcc_fraction.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+description: Ohio Federal CDCC Match Percentage
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/child/cdcc_fraction.yaml:1]:
```suggestion
description: Ohio matched the federal CDCC at this rate.
```

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/child/cdcc_fraction.yaml [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+values:
+  2021-01-01: 0.25
+metadata:
+  period: year
+  unit: /1
+  label: OH Child Care and Dependent Care Credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/child/cdcc_fraction.yaml:7]:
```suggestion
  label: Ohio Child Care and Dependent Care Credit match rate
```

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/age_threshold.yaml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+description: OH Senior Citizen Credit Age Threshold
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/age_threshold.yaml:1]:
```suggestion
description: Ohio provides the Senior Citizen Credit to filers at or above this age threshold.
```

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/age_threshold.yaml [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+values:
+  2021-01-01: 65
+metadata:
+  unit: years
+  period: year
+  label: OH Senior Citizen Credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/age_threshold.yaml:7]:
```suggestion
  label: Ohio Senior Citizen Credit age threshold
```

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/age_threshold.yaml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+description: OH Senior Citizen Credit Age Threshold
+values:
+  2021-01-01: 65
+metadata:
+  unit: years
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/age_threshold.yaml:5]:
```suggestion
  unit: year
```

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py [hunk 10]
```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class oh_senior_citizen_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio senior citizen credits"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py:7]:
```suggestion
    label = "Ohio senior citizen credit"
```

File: changelog_entry.yaml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+- bump: minor
+  changes:
+    added:
+    - Added Ohio Child Care and Dependent Care Credit
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Added Ohio Child Care and Dependent Care Credit.
```

File: changelog_entry.yaml [hunk 12]
```diff
@@ -0,0 +1,5 @@
...
+    - Added Ohio Senior Citizen Credit
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:5]:
```suggestion
    - Added Ohio Senior Citizen Credit.
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+    label = "Ohio senior citizen credits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=20",
+    )
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        agi_upper_limit = parameters(
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py:16]: Then access `p.x` in the `{oh,us}_eligible` statements below, directly. For this and other parameters.
```suggestion
        p =   parameters(period).gov.states.oh.tax.income.credits.child
```

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+        agi_partial_credit_limit = parameters(
+            period
+        ).gov.states.oh.tax.income.credits.child.agi_partial_credit_limit
+        oh_cdcc_fraction = parameters(
+            period
+        ).gov.states.oh.tax.income.credits.child.cdcc_fraction
+
+        agi = tax_unit("oh_agi", period)
+        us_cdcc = tax_unit("cdcc", period)
+
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py:28]: Please add comments

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_partial_credit_limit.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_partial_credit_limit.yaml:1]: Replace these three parameters with a single scale parameter:
* [-np.inf, 20_000) -> 1
* [20_000, 40_000) -> 0.25
* [40_000, np.inf] -> 0

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/agi_cap.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/agi_cap.yaml:1]: Rename to `agi_limit.yaml` for consistency with child credit

File: policyengine_us/tests/policy/baseline/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.yaml [hunk 15]
```diff
@@ -0,0 +1,39 @@
+- name: senior citizen test eligible
+  period: 2021
+  input:
+    state_code: OH
+    age: 65
+    oh_has_not_taken_oh_lump_sum_credits: 1
```
Comment (MaxGhenis) [inline policyengine_us/tests/policy/baseline/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.yaml:6]: should be bool

File: policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/agi_cap.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/oh/tax/income/credits/senior_citizen/agi_cap.yaml:1]: similarly replace this and credit_amount with a single scale parameter

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py [hunk 16]
```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class oh_child_dependent_care_credits(Variable):
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py:4]:
```suggestion
class oh_cdcc(Variable):
```

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio senior citizen credits"
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py:7]:
```suggestion
    label = "Ohio child and dependent care credit"
```

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py [hunk 18]
```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class oh_senior_citizen_credits(Variable):
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py:4]:
```suggestion
class oh_senior_citizen_credit(Variable):
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**